docs/modules/release-notes/pages/5-4-0.adoc

```diff
@@ -0,0 +1,248 @@
+Also removed the permissions for MongoDB connector.
+https://github.com/hazelcast/hazelcast/pull/25744[#25744], https://github.com/hazelcast/hazelcast/pull/25701[#25701]
+* Changed the return values for `getMethodName()` of the following message task classes:
+** `MapFetchKeysMessageTask`: `iterator` -> `SecurityInterceptorConstants.ITERATOR_FETCH_KEYS`
+** `MapFetchWithQueryMessageTask.`: `iterator` -> `SecurityInterceptorConstants.ITERATOR_FETCH_WITH_QUERY`
+** `MapProjectionWithPredicateMessageTask`: `projectWithPredicate` -> `SecurityInterceptorConstants.PROJECT`
+** `ScheduledExecutorGetAllScheduledMessageTask`: `getAllScheduled` -> `SecurityInterceptorConstants.GET_ALL_SCHEDULED_FUTURES`
+** `ScheduledExecutorSubmitToPartitionMessageTask`: `submitToPartition` -> `SecurityInterceptorConstants.SCHEDULE_ON_PARTITION`
+** `ScheduledExecutorSubmitToTargetMessageTask`: `submitToAddress` -> `SecurityInterceptorConstants.SCHEDULE_ON_MEMBER`
+** `ScheduledExecutorTaskGetResultFromPartitionMessageTask`: `getResultTimeout` -> `SecurityInterceptorConstants.GET_RESULT`
+** `ScheduledExecutorTaskGetResultFromTargetMessageTask`: `getResultTimeout` -> `SecurityInterceptorConstants.GET_RESULT`
+** `SetGetAllMessageTask`: `iterator` -> `SecurityInterceptorConstants.GET_ALL`
+** `TransactionalListAddMessageTask`: `set` -> `SecurityInterceptorConstants.ADD`
+** `TransactionalMultiMapRemoveEntryMessageTask`: `removeEntry` -> `SecurityInterceptorConstants.REMOVE`
++
+https://github.com/hazelcast/hazelcast/pull/25020[#25020]
+* Renamed the service port for Hazelcast clusters deployed in Kubernetes environments as `hazelcast`.
+Previously, the name was `hazelcast-service-port` causing the member auto-discovery (for embedded deployments) to fail.
+https://github.com/hazelcast/hazelcast/pull/24834[#24834], https://github.com/hazelcast/hazelcast/issues/24705[#24705], https://github.com/hazelcast/hazelcast/issues/24688[#24688]
+* The connector for Elasticsearch 6 does not exist anymore since the Elastichsearch 6 module has been removed from the Hazelcast distributions.
+https://github.com/hazelcast/hazelcast/pull/24734[#24734]
+* Fixed an issue where Hazelcast was not fast-failing when a member is started with a blank public address. This has been fixed by introducing a configuration validation
+which may break the existing invalid configurations.
+https://github.com/hazelcast/hazelcast/pull/24729[#24729]
+
+== Enhancements
+
+=== Cloud
+
+* ???
+* [.enterprise]*Enterprise* ???
+
+=== Storage
+
+* Tiered Store can now be used for map loaders. See ???.
+https://github.com/hazelcast/hazelcast/pull/24827[#24827]
+
+=== Distribution
+
+* ???
+
+=== Networking
+
+* Added the ability to evaluate priorities while picking member addresses; when interfaces are not configured, the strategy now does not use the first possible IP address, but evaluates all and pick one with the highest priority (being an IPv6 address).
```

Comment:
when interfaces are not configured, the first possible IP address is no longer used. Instead, all addresses are evaluated and the one with the highest priority (IPv6 address) is selected.